es/09_4_Codificando_una_P2PKH.md

```diff
@@ -0,0 +1,390 @@
+OP_CHECKSIG                                                        |                           41d83eaffbf80f82dee4c152de59a38ffd0b6021
+                                                                   | 0315a0aeb37634a71ede72d903acae4c6efa77f3423dcbcd6de3e13d9fd989438b
+                                                                   | 304402201cc39005b076cb06534cd084fcc522e7bf937c4c9654c1c9dfba68b...
+                                                                   | 
+                                                                   | 
+                                                                   |  
+                                                                   |  
+                                                                   |  
+                                                                   |       
+                                                                   |   
+                                                                   | 
+                                                                   |
+                                                                   | 
+#0005 OP_EQUALVERIFY
+btcdeb> step
+		<> POP  stack
+		<> POP  stack
+		<> PUSH stack 01
+		<> POP  stack
+script                                                             |                                                             stack 
+-------------------------------------------------------------------+-------------------------------------------------------------------
+OP_CHECKSIG                                                        | 0315a0aeb37634a71ede72d903acae4c6efa77f3423dcbcd6de3e13d9fd989438b
+                                                                   | 304402201cc39005b076cb06534cd084fcc522e7bf937c4c9654c1c9dfba68b...
+                                                                   | 
+                                                                   | and_v(
+                                                                   |     sig(304402201cc39005b076cb06534cd084fcc522e7bf937c4c9654c1c...
+                                                                   |     and_v(
+                                                                   |         pk(0315a0aeb37634a71ede72d903acae4c6efa77f3423dcbcd6de3...
+                                                                   |         c:pk_h(030500000000000000000000000000000000000000000000...
+                                                                   |     )
+                                                                   | 
+                                                                   | )
+                                                                   | 
+			
+```
+En este punto, todo lo que se requiere es el `OP_CHECKSIG`:
+```
+#0006 OP_CHECKSIG
+btcdeb> step
+error: Signature is found in scriptCode
+```
```

Comment:
Missing fragment: 
``(Unfortunately this checking may or may not be working at any point due to vagaries of the Bitcoin Core and btcdeb code.)``

``As is shown, a P2PKH is quite simple: its protection comes about through the strength of its cryptography.``